packages/manager/apps/cloud-connect/src/app.module.js

```diff
@@ -0,0 +1,8 @@
+import 'script-loader!jquery'; // eslint-disable-line
+import '@ovh-ux/manager-cloud-connect';
+import { Environment } from '@ovh-ux/manager-config';
+import angular from 'angular';
+
+Environment.setRegion(__WEBPACK_REGION__);
```

Comment:
i don't think setting the region using webpack is need anymore, calling bootstrapApplication is enough